products/models.py

```diff
@@ -0,0 +1,70 @@
+from django.db import models
+
+class Menu(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta : 
+        db_table = 'menus'
+
+class Category(models.Model) : 
+    name = models.CharField(max_length=50)
+    menu = models.ForeignKey('Menu', on_delete=models.CASCADE)
+
+    class Meta : 
+        db_table = 'categories'
+
+class Subcategory(models.Model) :
+    name = models.CharField(max_length=100)
+    category = models.ForeignKey('Category', on_delete=models.CASCADE)
+
+    class Meta : 
+        db_table = 'subcategories'
+
+class Product(models.Model) : 
+    subcategory = models.ForeignKey('Subcategory', on_delete=models.CASCADE)
+    name = models.CharField(max_length=500)
+    author = models.CharField(max_length=500)
+    publisher = models.CharField(max_length=100)
+    date_published = models.DateField()
+    float = models.DecimalField(max_digits=7, decimal_places=2)
+    description = models.TextField()
+    index = models.TextField()
+    thumbnail_img_url = models.CharField(max_length=2000)
```

Comment:
컬럼명은 되도록이면 줄임말없이 사용해주세요!
```suggestion
    thumbnail_image_url = models.CharField(max_length=2000)
```

Reply:
수정 완료하였습니다 😊